core/src/main/resources/application.yml

```diff
@@ -108,7 +108,7 @@ altus:
     caller: cloudbreak
 
 cb:
-  enabledplatforms: AZURE,AWS,GCP
```

Comment:
Wasn't this intended for the Environment app.yml

Won't this make MOCK visible in prod environments?

Reply:
I can revert that change, it was introduced because previously the empty list value for `cb.enabledplatforms` was resulted in enabling all the supported providers and most of the developers used that. So basically it's just mitigates the basic setup without overriding the mentioned platform properties...
In the description I also mentioned that all the CSI repo overrides have been checked and every manowar env has it's explicit set of the supported cloud providers.